checks.go

```diff
@@ -216,7 +222,11 @@ func (c cond) FileContains() (bool, error) {
 	}
 	found := false
 	for _, line := range strings.Split(fileContent, "\n") {
-		found, err = regexp.Match(c.Value, []byte(line))
+		if c.Regex {
+			found, err = regexp.Match(c.Value, []byte(line))
+		} else {
+			found = strings.Contains(line, c.Value)
+		}
 		if err != nil {
```

Comment:
Same here, I think err check should be moved up to under regex match

Reply:
so just make sure each error != nil is always directly under the error causer?